rosdep/python.yaml

```
@@ -6691,6 +6691,10 @@ python3-pyqtgraph:
   fedora: [python3-pyqtgraph]
   gentoo: [dev-python/pyqtgraph]
   ubuntu: [python3-pyqtgraph]
+python3-pyrr:
```

Since this is a pip-only package, please make this:

```suggestion
python3-pyrr-pip:
```